docs/UserGuide.md

```diff
@@ -78,6 +122,18 @@ This tool performs a diff of a CR against the reference. It does not have access
 is available through the cluster's API. This places validation of the configuration against underlying platform
 hardware, os configuratoin, etc (unless available through the api) out of scope.
 
+###  States of a Reference Configuration CR after running the tool 
+
+After running `kubectl cluster-compare` tool, the reference configuration CRs will be identified in one of the following states: missing and present in the live cluster.
+1. Missing CR. A `missing` reference CR can be in one of the following states: 
```

Comment:
I believe the tool will only report on missing CRs if they are required, ie optional CRs aren't listed in the output as missing (because they are optional). The exception to this is that an optional component may have required CRs defined within it. That means that if any part of that optional component is included in the cluster's config the tool will report out missing status for the required parts of that (optional) component.

Maybe it would be helpful to put this section in context of the tool's output?

Reply:
Done.